website/docs/reference/resource-properties/unit-tests.md

```diff
@@ -58,100 +58,16 @@ unit_tests:
 Unit tests are currently limited to testing SQL models and only models in your current project. 
 
 ### Versions
-If your model has multiple versions, the default unit test will run on *all* versions of your model. To specify version(s) of your model to unit test, use `include` or `exclude` for the desired versions in your model versions config:
 
-```yaml
-
-# my test_is_valid_email_address unit test will run on all versions of my_model
-unit_tests:
-  - name: test_is_valid_email_address
-    model: my_model
-    ...
-            
-# my test_is_valid_email_address unit test will run on ONLY version 2 of my_model
-unit_tests:
-  - name: test_is_valid_email_address 
-    model: my_model 
-      versions:
-        include: 
-          - 2
-    ...
-            
-# my test_is_valid_email_address unit test will run on all versions EXCEPT 1 of my_model
-unit_tests:
-  - name: test_is_valid_email_address
-    model: my_model 
-      versions:
-        exclude: 
-          - 1
-    ...
-
-```
+See [Unit testing versioned SQL models](/reference/resource-properties/unit-testing-versions)
 
 ### Format
```

Comment:
Do we still want these headings if the content is on another page?

Reply:
I was thinking leave them there since people had already seen the doc and maybe wondered where those sections went, but the views are low enough that I think we'll be ok. I'll make the change.